src/components/Drawer.js

```diff
@@ -222,15 +240,15 @@ class Drawer extends React.Component {
                     </Button>
                   }
                 </span>
-                <span style={styles.title}>
+                <h1 style={styles.title}>
                   {this.props.title}
-                </span>
+                </h1>
```

Comment:
the `<h1>` here is tough - I can see us running into potential issues in the future where the title should not be an `<h1>`... I think what could be a good solution would be to make the title a prop that takes a tiny functional component, then the user could do something like this: 

```
<Drawer title={() => <h1>If I want an h1 <h1>} /> 
or 
<Drawer title={() => <h2>If I want an h2 <h2>} /> 
```
I think you would just have to do something with the styling to normalize the styles no matter what tag the user used here...

Reply:
ooooh what about this: 
```
<Drawer title={({ style }) => <h1 style={style}>Here we go</h1>
```

and then within the Drawer the code would look like this: 
```
<this.props.title style={{ ...here youd put the styles you need to normalize }} /> 
```

Reply:
I considered what you suggested when I first started working on this but settled on the h1 directly because I couldn't think of a single use case where a drawer title shouldn't be an h1.  It describes the drawers contents.  It's also easier to remember that children of the drawer should use h2 or less when I know that the title is always an h1.

Reply:
alright - also a lot of the research i've done lately says multiple H1 tags on a page is ok, so I suppose we could be fine even if there was <h1> content within the drawer

Reply:
If you want to chat in person about it I'm more than happy to.

Reply:
no we're on the same page 👍 i was just thinking out loud with my last comment